.github/workflows/bump_version.yml

```diff
@@ -0,0 +1,84 @@
+        type: string
+
+jobs:
+  bump-version:
+    runs-on: ubuntu-20.04
+    permissions:
+      id-token: write
+      contents: write
+      pull-requests: write
+      checks: write
+    env:
+      NEW_SEMGREP_VERSION: ${{ github.event.inputs.version }}
+      ACTOR: ${{ github.actor }}
+      RUN_ID: ${{ github.run_id }}
+      RUN_ATTEMPT: ${{ github.run_attempt }}
+    steps:
+      - name: Get JWT for r2c-argo GitHub App
+        id: jwt
+        uses: docker://public.ecr.aws/y9k7q4m1/devops/cicd:latest
+        env:
+          EXPIRATION: 600 # seconds
+          ISSUER: 132926 # r2c-argo GitHub App id
+          PRIVATE_KEY: ${{ secrets.R2C_ARGO_PRIVATE_KEY }}
+      - name: Get token for r2c-argo GitHub App
+        id: token
+        run: |
+          # r2c-argo GitHub App installation id for semgrep-app-kubernetes repo (18942934)
+          TOKEN="$(curl -X POST \
+          -H "Authorization: Bearer ${{ steps.jwt.outputs.jwt }}" \
+          -H "Accept: application/vnd.github.v3+json" \
+          https://api.github.com/app/installations/18942934/access_tokens | \
+          jq -r .token)"
+          echo "::add-mask::$TOKEN"
+          echo "::set-output name=token::$TOKEN"
+      - uses: actions/checkout@v3
+        with:
+          submodules: true
+          token: ${{ steps.token.outputs.token }}
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.9"
+      - name: Bump semgrep version
+        run: |
+          scripts/bump-version "$NEW_SEMGREP_VERSION"
+      - name: Commit and push
+        id: commit
+        run: |
+          git config user.name ${ACTOR}
+          git config user.email ${ACTOR}@users.noreply.github.com
+          BRANCH="gha/bump-version-${NEW_SEMGREP_VERSION}-${RUN_ID}-${RUN_ATTEMPT}"
+          SUBJECT="Bump semgrep to ${NEW_SEMGREP_VERSION}"
+          git checkout -b $BRANCH
+          git add .
+          git commit -m "$SUBJECT"
+          git push --set-upstream origin $BRANCH
+          echo "::set-output name=branch::$BRANCH"
+          echo "::set-output name=subject::$SUBJECT"
+      - name: Create PR
+        id: open-pr
+        env:
+          SOURCE: "${{ steps.commit.outputs.branch }}"
+          TARGET: "${{ github.event.repository.default_branch }}"
+          TITLE: "chore: Release Version ${{ inputs.version }}"
+          GITHUB_TOKEN: ${{ steps.token.outputs.token }}
+        run: |
+          # check if the branch already has a pull request open
+          if gh pr list --head ${SOURCE} | grep -vq "no pull requests"; then
+              # pull request already open
+              echo "pull request from SOURCE ${SOURCE} to TARGET ${TARGET} is already open";
+              echo "cancelling release"
+              exit 1
+          fi
+          # open new pull request with the body of from the local template.
+          res=$(gh pr create --title "${TITLE}" --body "Bump Semgrep Version to ${NEW_SEMGREP_VERSION}" \
```

Comment:
can we also just remove the output capture here? makes it easier to debug if anything goes wrong.

Reply:
Good call! Updated in latest commit